### PR TITLE
Release 4.19.0 ALPHA

### PR DESCRIPTION
**Pull request by dmnks**
Tweak a few changelog entries as well.

### CODE REVIEW

**Comment by dmnks** [comment]
Made a fixup based on our chat discussion today.

File: wiki/Releases/4.19.0.md [hunk 1]
```diff
@@ -43,7 +38,7 @@ title: rpm.org - Releases
 
 #### Buildroot policies
 * Fix a race condition in `brp-strip` (RhBug:1959049)
-* `check-buildroot` hardening fixes
+* Hardening fixes in `check-buildroot`
 
 #### Generators
 * Generate user/group provides from packaged `/etc/passwd` and `/etc/group` or `sysusers.d(5)` (#1032)
```
Comment (dmnks) [inline wiki/Releases/4.19.0.md]: Still not sure whether to keep this one here :smile:

Comment (pmatilai) [reply wiki/Releases/4.19.0.md]: Oh, I think it's quite important actually. People need to be made aware of new dependencies being generated. Even more important is to note that rpm now adds user/group *requires* for non-root owned files! Which will require distro-level coordination / bootstrapping to sort out. That would actually be good to mention in the compatibility notes.

Comment (dmnks) [reply wiki/Releases/4.19.0.md]: > Oh, I think it's quite important actually. People need to be made aware of new dependencies being generated. 

OK

> Even more important is to note that rpm now adds user/group _requires_ for non-root owned files!

Oh, right, I removed that line in this PR actually (as a misguided attempt at trimming down the content), will add it back :smile: 

> Which will require distro-level coordination / bootstrapping to sort out. That would actually be good to mention in the compatibility notes.

OK, will add a note there, too.

**Comment by dmnks** [comment]
Updated with the suggestions above. SHA256 checksum updated for the fixed tarball as well.

**Comment by dmnks** [comment]
... aaaand merge :smile:

**Pull request merged**